package.json

```diff
@@ -14,5 +14,8 @@
     "bottleneck": "^2.19.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1"
+  },
+  "devDependencies": {
+    "jest": "^26.6.3"
   }
```

Comment:
This will fix your testing =) 
<img src="https://user-images.githubusercontent.com/1356595/105807598-2c032b80-5f5b-11eb-81fe-596daadd3a2a.png" width=20% height=20%>

Reply:
🙌  thank you!!!